public/app/features/plugins/admin/api.ts

```diff
@@ -108,8 +109,9 @@ async function getLocalPluginReadme(id: string): Promise<string> {
 }
 
 export async function getLocalPlugins(): Promise<LocalPlugin[]> {
-  const installed = await getBackendSrv().get(`${API_ROOT}`, { embedded: 0 });
-  return installed;
+  const localPlugins: LocalPlugin[] = await getBackendSrv().get(`${API_ROOT}`, { embedded: 0 });
+
+  return localPlugins.filter(Boolean).filter(isLocalPluginVisible);
```

Comment:
How come we need to do `.filter(Boolean)` here?

Reply:
Thanks for spotting this mate! 🙏 
Probably something that I didn't manage to clean up during the rebase.

Reply:
Removed in https://github.com/grafana/grafana/pull/41679/commits/179fdc88bdc69d6e164c4022846e215d0a5b7ad5 👍